Source/WebCore/page/NavigatorLoginStatus.cpp

```diff
@@ -26,11 +26,22 @@
 #include "config.h"
 #include "NavigatorLoginStatus.h"
 
+#include "Chrome.h"
+#include "ChromeClient.h"
 #include "JSDOMPromiseDeferred.h"
 #include "Navigator.h"
+#include "Page.h"
+#include "WebCore/RegistrableDomain.h"
+#include "dom/Document.h"
+#include "dom/DocumentStorageAccess.h"
+#include <wtf/StdLibExtras.h>
+#include <wtf/text/WTFString.h>
```

Comment:
Many of these includes are likely not needed.

Reply:
But I need Document to get access to frame and page, no?
```
Document* document = m_navigator.document();
RefPtr frame = document->frame();
 page = frame->page();
```

Reply:
Right, you probably need document, but you can remove the `dom/` and change the include to just #include "Document.h"`.